### PR TITLE
maestral: 1.5.3 -> 1.6.1

### PR DESCRIPTION
**Pull request by peterhoeg**
###### Description of changes

Upstream update.

There's a bit of noise from nixpkgs-fmt - apologies.

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/maestral/default.nix [hunk 1]
```diff
@@ -43,14 +56,24 @@ buildPythonPackage rec {
...
-    "--prefix" "PYTHONPATH" ":" "${lib.concatStringsSep ":" (map (p: p + "/lib/${python.libPrefix}/site-packages") (python.pkgs.requiredPythonModules propagatedBuildInputs))}"
-    "--prefix" "PYTHONPATH" ":" "$out/lib/${python.libPrefix}/site-packages"
+    "--prefix"
+    "PYTHONPATH"
+    ":"
+    "${lib.concatStringsSep ":" (map (p: p + "/lib/${python.libPrefix}/site-packages") (python.pkgs.requiredPythonModules propagatedBuildInputs))}"
+    "--prefix"
+    "PYTHONPATH"
+    ":"
+    "$out/lib/${python.libPrefix}/site-packages"
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/maestral/default.nix:66]:
```suggestion
    "--prefix PYTHONPATH : ${lib.concatStringsSep : (map (p: p + "/lib/${python.libPrefix}/site-packages") (python.pkgs.requiredPythonModules propagatedBuildInputs))}"
    "--prefix PYTHONPATH : $out/lib/${python.libPrefix}/site-packages"
```

**Comment by peterhoeg** [comment]
It's auto-formatted by nixpkgs-fmt. It's not ideal.

**Comment by peterhoeg** [comment]
It's not nice, I agree. But manually messing around with formatting isn't either. And especially since this will happen again the next time anyone edits this file.

**Comment by SFrijters** [comment]
It builds, but doesn't seem to play nicely with maestral-gui (maestral-qt) which is stuck at 1.5.3 until we have Qt6 available:
```
Traceback (most recent call last):
  File "/nix/store/f5w9rnr72my31nas8mvms0pqqvjzzx3q-maestral-qt-1.5.3/bin/.maestral_qt-wrapped", line 6, in <module>
    from maestral_qt.__main__ import main
  File "/nix/store/f5w9rnr72my31nas8mvms0pqqvjzzx3q-maestral-qt-1.5.3/lib/python3.9/site-packages/maestral_qt/__main__.py", line 7, in <module>
    from maestral_qt.main import run
  File "/nix/store/f5w9rnr72my31nas8mvms0pqqvjzzx3q-maestral-qt-1.5.3/lib/python3.9/site-packages/maestral_qt/main.py", line 37, in <module>
    from maestral.errors import KeyringAccessError
ModuleNotFoundError: No module named 'maestral.errors'
```

File: pkgs/development/python-modules/maestral/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/development/python-modules/maestral/default.nix:66]: but that should work the same and avoids the formatting mess

**Comment by peterhoeg** [comment]
Crap, I only use the CLI version.

https://github.com/samschott/maestral/issues/655

**Comment by peterhoeg** [comment]
Closing this for now until upstream adds back qt5 support to avoid breaking the GUI version.

**Comment by peterhoeg** [comment]
@GrahamcOfBorg build maestral maestral-qt

**Pull request closed without merging**